src/app/components/top/top.component.ts

```diff
@@ -52,4 +76,16 @@ export class TopComponent {
     this.onOpenInExplorer.emit(true);
   }
 
+  setStarRating(rating: StarRating): void {
```

Comment:
This is duplicating `setStarRating` in the `meta-component` - so it's a good time to create a new component that will have this functionality 👍 and we can then remove this method from both here and `meta-component` 👍